### ISSUE TITLE
BACKUP VR of VPC  Enters MASTER state and then BACKUP state when add a new VPC tier

### ISSUE THREAD

**Issue by weizhouapache** [apache/cloudstack#8108]
Steps to reproduce the issue

- create  a vpc vpc-001 with redundant VRs
- create vpc tier vpc-001-001, create a vm
- create vpc tier vpc-001-002, create a vm

check /var/log/messages, there are errors like
```
Oct 17 07:40:13 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:14 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:15 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:16 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:16 r-186-VM Keepalived_vrrp[1230230]: (inside_network) Entering MASTER STATE
Oct 17 07:40:22 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 377 already processed. Packet dropped. Local(377)
Oct 17 07:40:23 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 378 already processed. Packet dropped. Local(378)
Oct 17 07:40:24 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 379 already processed. Packet dropped. Local(379)
Oct 17 07:40:25 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 380 already processed. Packet dropped. Local(380)
Oct 17 07:40:26 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 381 already processed. Packet dropped. Local(381)
Oct 17 07:40:27 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 382 already processed. Packet dropped. Local(382)
Oct 17 07:40:28 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 383 already processed. Packet dropped. Local(383)
Oct 17 07:40:29 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 384 already processed. Packet dropped. Local(384)
Oct 17 07:40:42 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 385 already processed. Packet dropped. Local(385)
Oct 17 07:40:43 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 386 already processed. Packet dropped. Local(386)
Oct 17 07:40:44 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 387 already processed. Packet dropped. Local(387)
Oct 17 07:40:45 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 388 already processed. Packet dropped. Local(388)
Oct 17 07:40:46 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 389 already processed. Packet dropped. Local(389)
Oct 17 07:40:47 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 390 already processed. Packet dropped. Local(390)
Oct 17 07:40:48 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 391 already processed. Packet dropped. Local(391)
Oct 17 07:40:49 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 392 already processed. Packet dropped. Local(392)
Oct 17 07:40:50 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 393 already processed. Packet dropped. Local(393)
Oct 17 07:40:51 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 394 already processed. Packet dropped. Local(394)
Oct 17 07:40:52 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 395 already processed. Packet dropped. Local(395)
Oct 17 07:40:53 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 396 already processed. Packet dropped. Local(396)
Oct 17 07:40:54 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 397 already processed. Packet dropped. Local(397)
Oct 17 07:40:55 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 398 already processed. Packet dropped. Local(398)
Oct 17 07:40:56 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 399 already processed. Packet dropped. Local(399)
Oct 17 07:40:57 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:58 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:40:59 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:00 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:01 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:02 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:03 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:04 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:05 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:06 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:07 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:08 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:09 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:10 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:11 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:12 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:13 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:14 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:15 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:16 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:17 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:18 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:19 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:20 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 423 already processed. Packet dropped. Local(423)
Oct 17 07:41:21 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 424 already processed. Packet dropped. Local(424)
Oct 17 07:41:22 r-186-VM Keepalived_vrrp[1230230]: (inside_network) IPSEC-AH : sequence number 425 already processed. Packet dropped. Local(425)
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: (inside_network) received an unexpected ip number count 2, expected 1!
Oct 17 07:41:23 r-186-VM Keepalived[1230226]: Reloading ...
Oct 17 07:41:23 r-186-VM Keepalived[1230226]: Opening file '/etc/keepalived/keepalived.conf'.
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: Reloading
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: Opening file '/etc/keepalived/keepalived.conf'.
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: VRRP_Script(heartbeat) considered successful on reload
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: SECURITY VIOLATION - scripts are being executed but script_security not enabled.
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: (inside_network) Master received advert from 172.16.1.177 with same priority 100 but higher IP address than ours
Oct 17 07:41:23 r-186-VM Keepalived_vrrp[1230230]: (inside_network) Entering BACKUP STATE
```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~

~~~

##### CLOUDSTACK VERSION

~~~
most recent versions
~~~

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by DaanHoogland** [comment]
@weizhouapache have you seen any disfunction because of this? I'm meaning to set a severity, that's why I ask.

**Comment by vishesh92** [comment]
Working on a fix in this PR: https://github.com/apache/cloudstack/pull/9251

**Comment by DaanHoogland** [comment]
fixed in #9251
